### PR TITLE
Enable cluster memory manager by default

### PR DESCRIPTION
**Pull request by cberner**
Also add documentation

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -157,6 +163,12 @@ These properties require some explanation:
   queries that can be run, while setting it too high will cause the JVM
   to run out of memory.
 
+* ``query.max-memory=50GB``:
+  The maximum amount of distributed memory that a query may use.
+
+* ``query.max-memory-per-node=1GB``:
+  The maximum amount of memory that a query may use on any one machine.
```
Comment (dain) [inline presto-docs/src/main/sphinx/installation/deployment.rst]: should we document that they need more memory than this in the heap, or maybe fail to start on a really bad number

**Comment by dain** [comment]
looks good

File: presto-docs/src/main/sphinx/release/release-0.113.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Cluster Resource Management
+---------------------------
+
+The cluster resource manager announced in :doc:`/release/release-0.103` is now enabled by default.
+You can disable it with the ``experimental.cluster-memory-manager-enabled`` flag.
+Memory limits can now be configured via ``query.max-memory`` which controls the total distributed
+memory a query may use and ``query.max-memory-per-node`` which limits the amount
+of memory a query may use on any one node. On each worker, the
+``resources.reserved-system-memory`` flags controls how much memory is reserved
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.113.rst]: flags -> flag

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.113.rst]: flags -> config property

**Pull request merged**